airflow/README.md

```diff
@@ -155,11 +155,11 @@ Connect Airflow to DogStatsD (included in the Datadog Agent) by using the Airflo
            name: "airflow.dagrun.schedule_delay"
            tags:
              dag_id: "$1"
-         - match: 'scheduler.tasks.running'
+         - match: 'airflow.scheduler.tasks.running'
            name: "airflow.scheduler.tasks.running"
-         - match: 'scheduler.tasks.starving'
+         - match: 'airflow.scheduler.tasks.starving'
            name: "airflow.scheduler.tasks.starving"
-         - match: sla_email_notification_failure
+         - match: airflow.sla_email_notification_failure
```

Comment:
should this match also be wrapped in quotes?
```suggestion
         - match: 'airflow.sla_email_notification_failure'
```

Reply:
good catch, thank you, updated!